nx/include/switch/applets/error.h

```diff
@@ -0,0 +1,74 @@
+/**
+ * @file error.h
+ * @brief Wrapper for using the error LibraryApplet.
+ * @author StuntHacks
+ * @copyright libnx Authors
+ */
+#pragma once
+#include "../types.h"
+#include "../services/applet.h"
+
+/// Error configuration struct.
```

Comment:
Do this properly with the actual storage struct, not a wrapper.

Reply:
See 7e42dfe (also f816ac5).